cmd/cli/root.go

```diff
@@ -44,6 +44,18 @@ func Execute() {
 func getKubernetesClient(kubePath string) (kubernetes.Interface, error) {
 	var config *rest.Config
 
+        if _, inCluster := os.LookupEnv("KUBERNETES_SERVICE_HOST"); inCluster == true {
```

Comment:
Is `KUBERNETES_SERVICE_HOST` an arbitrary name or is it commonly used else where?
Wondering if we should use a different variable name here that may be more commonly used for this purpose / easier to understand it's purpose

Reply:
Oh, I see - this is a variable that is exported by default on pods. 
Is the purpose here to use the CLI from inside a pod vs. using aws-auth as a library?